Product/Production/Gateway/DocumentQuery_30/src/main/resources/docquery/_30/webservices.xml

```diff
@@ -52,10 +52,10 @@
             <ref bean="TransactionHandler" />
         </jaxws:handlers>
     </jaxws:endpoint>
-    
+
     <jaxws:endpoint xmlns:ndq="urn:gov:hhs:fha:nhinc:dq:nhindeferredresultsecured" id="NhinDocQueryDeferredResponseSecured"
         address="/NhinService/DocumentQueryDeferredResponse" serviceName="ndq:NhinDocQueryDeferredResponseSecured"
-        endpointName="ndq:NhinDocQueryDeferredRequestSecuredPortSoap" implementor="gov.hhs.fha.nhinc.docquery.deferred.nhin.NhinDeferredResultsOption"
+        endpointName="ndq:NhinDocQueryDeferredRequestSecuredPortSoap" implementor="#nhinDeferredResults"
```

Comment:
Should have "implementorClass" set to the interface to explicitly state the class type for injection